pysteps/blending/linear_blending.py

```diff
@@ -166,45 +173,137 @@ def forecast(
         )
 
         # Initialise output
-        R_blended = np.zeros_like(precip_nowcast)
+        precip_blended = np.zeros_like(precip_nowcast)
 
         # Calculate the weights
         for i in range(timesteps):
             # Calculate what time we are at
             t = (i + 1) * timestep
 
+            if n_ens_members_max == 1:
+                ref_dim = 0
+            else:
+                ref_dim = 1
+
+            # compute the slice indices
+            def _get_slice(n_dims, ref_dim, ref_id):
+                """source: https://stackoverflow.com/a/24399139/4222370"""
+                slc = [slice(None)] * n_dims
+                slc[ref_dim] = ref_id
+                return tuple(slc)
```

Comment:
Maybe nicer to put this function at the end of the code (similar to the steps.py code).